packages/oas-utils/src/entities/spec/collection.ts

```diff
@@ -53,6 +53,8 @@ export const extendedCollectionSchema = z.object({
   tags: nanoidSchema.array().default([]),
   /** List of requests without tags and top level tag "folders" */
   children: nanoidSchema.array().default([]),
+  /** An icon representing the collection */
+  icon: z.string().default('interface-content-folder'),
```

Comment:
@amritk @marclave  should this be `icon` or `x-scalar-icon`? I'm not sure how this schema relates to the open api spec 🤔

Reply:
You probably want to move it up to the `oasCollectionSchema` and call it `x-scalar-icon` but putting it down there isn't the end of the world. If you leave it in the bottom there as icon we will just need to convert it on the way in/out. Putting it above in the openapi spec section saves the conversion

Reply:
usually we have a spec compliant schema which starts with `oas` and an extended schema which is the non spec compliant stuff